src/plugin/code.js

```diff
@@ -181,6 +184,64 @@ figma.ui.onmessage = async message => {
     clearRectangleNode();
 
     renderDifferenceRectangle(differences, modifiedFrames);
+
+    const startNodeId = differenceRectangleIdList[0];
+    const startNode = figma.getNodeById(startNodeId);
+
+    figma.viewport.scrollAndZoomIntoView([startNode]);
+    figma.currentPage.selection = [startNode];
+  }
+
+  if (message.type === "PREV_DIFFERENCE_RECTANGLE") {
+    const isSelectedNode = figma.currentPage.selection[0];
+
+    if (
+      isSelectedNode &&
+      differenceRectangleIdList.includes(isSelectedNode.id)
+    ) {
+      const currentNodeId = figma.currentPage.selection[0].id;
+      const currentNodeIndex = differenceRectangleIdList.indexOf(currentNodeId);
+      const prevNodeIndex =
+        (currentNodeIndex - 1) % differenceRectangleIdList.length;
+      const prevNodeId = differenceRectangleIdList[prevNodeIndex];
+      const prevNode = figma.getNodeById(prevNodeId);
+
+      figma.viewport.scrollAndZoomIntoView([prevNode]);
+      figma.currentPage.selection = [prevNode];
+    } else {
+      const startNodeId = differenceRectangleIdList[0];
+      const startNode = figma.getNodeById(startNodeId);
+
+      figma.viewport.scrollAndZoomIntoView([startNode]);
+      figma.currentPage.selection = [startNode];
+    }
+    figma.notify(`이전 변경 사항이 선택 되었습니다.`);
```

Comment:
```suggestion
    figma.notify(`이전 변경 사항이 선택 되었습니다.`);
```
P1: 요 부분도 개행 부탁드립니다 :) 

```suggestion
    }
    
    figma.notify(`이전 변경 사항이 선택 되었습니다.`);
```